docs/reference.md

```diff
@@ -1826,35 +1828,59 @@ Allowed values:
 
 Executes an HTTP request.
 
-_nb: Only APIs that return a JSON response are supported.
-
 Key | Type | Required |
 ------------ | -------| --- |
 [url](#url) | String | Yes |
 [method](#method) | String | Yes |
-[body](#body) | String | No |
+[body](#body) | Object/String | No |
 [headers](#headers) | String | No |
 
+_nb: For multipart/form-data content type requests, the body should be provided as a key/value object. For other content types, it can be provided as String in JSON format.
+
+Output | Type |
+----|----|
+body | String containing the response body
```

Comment:
To be updated?

Reply:
What should I set?  This didn't change, outputs.body still has the response body, in a String format.

Reply:
That's me, mixing json() usage and the raw body